### PR TITLE
Purge links to oasis.

### PR DESCRIPTION
**Pull request by zenhack**
This removes most references to oasis. There are a few exceptions:

* The /get page is unchanged, since I have another pr out to just
  redirect that whole page to /install.
* The /apps path is unchanged. We should probably just kill that and
  redirect it to the app market.
* The sign-in buttons are still there; per @ocdtrekkie's suggestion,
  they should not be removed until oasis is no longer offering users
  the ability to download their old data.

### CODE REVIEW

**Comment by ocdtrekkie** [comment]
I think Web Shell (which is kinda neat, but useless) and Mailpile (which is horrifically old and probably very risky to use for your mail) are the only two apps that never made it to the modern apps list. I would not be too sad about /apps going away.

I would be somewhat terrified if someone was using a version of Sandstorm that still linked to it.

**Comment by zenhack** [comment]
Looking at the diff I realize my editor stripped off some trailing whitespace too. Sorry it's a bit messy.

File: _includes/sidebar.html [hunk 1]
```diff
@@ -9,10 +9,9 @@
   <div class="more">
     <a href="/get">
       Sandstorm is a self-hostable web productivity suite.
-      <span class="signup" href="https://oasis.sandstorm.io">Sign up</span>
-      <span class="selfhost" href="/install">or host your own</span>
+      <span class="selfhost" href="/install">Host your own</span>
```
Comment (ocdtrekkie) [inline _includes/sidebar.html]: I am tempted to suggest you abuse class=signup here. If you look at https://sandstorm.io/news/ you'll notice that the signup gets a nice purple button, and selfhost is just a web link. I know which one I think will look better there.

Comment (zenhack) [reply _includes/sidebar.html]: Or even better, I could just change the name and not be abusive about it. It looks like nothing else uses these styles anyway. Done.

**Pull request merged**